site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.tsx

```diff
@@ -115,7 +137,14 @@ const CourseList: FC<{ courses: string[]; takenCourseIDs: CompletedCourseSet }>
       className={'group-courses' + (isMobile ? ' disabled' : '')}
     >
       {courses.map((c, i) => (
-        <CourseTile courseID={c} key={c} dragTimestamp={timestamps[i]} taken={c in takenCourseIDs} />
+        <CourseTile
+          courseID={c}
+          key={c}
+          dragTimestamp={timestamps[i]}
+          taken={c in takenCourseIDs}
+          transferredByAP={c in transferredCourseIDs && transferredCourseIDs[c].transferType === 'AP'}
+          transferredByCourse={c in transferredCourseIDs && transferredCourseIDs[c].transferType === 'Course'}
```

Comment:
Use one prop instead of 3
```suggestion
          completedBy={c in takenCourseIDs ? (takenCourseIDs[c].transferType ?? 'roadmap') : null}
```

Then change the prop type to be `TransferredCourseWithType['transferType'] | 'roadmap' | null`